client/src/components/home/Home.tsx

```diff
@@ -1,27 +1,29 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { DefaultButton, PartialTheme, StackItem, Theme } from '@fluentui/react';
+import { DefaultButton, ImageFit, PartialTheme, StackItem, Theme } from '@fluentui/react';
 import { Stack, Text, Image } from '@fluentui/react';
```

Comment:
Lets rename this file, there is another Home.tsx and it might get confusing, we can rename this to HomeComponent.tsx

Reply:
have changed this and updated references to it and pushed a fix!